docs/ops/detection/ExperimentalDetectronGenerateProposalsSingleImage_8.md

```diff
@@ -0,0 +1,156 @@
+# ExperimentalDetectronGenerateProposalsSingleImage {#openvino_docs_ops_detection_ExperimentalDetectronGenerateProposalsSingleImage_8}
+
+**Versioned name**: *ExperimentalDetectronGenerateProposalsSingleImage-8*
+
+**Category**: *Object detection*
+
+**Short description**: The *ExperimentalDetectronGenerateProposalsSingleImage* operation computes ROIs and their scores
+based on input data.
+
+**Detailed description**: The operation performs the following steps:
+
+1.  Transposes and reshapes predicted bounding boxes deltas, variances and scores to get them into the same order as the
+anchors.
+2.  Transforms anchors into proposals using deltas, variances and clips proposals to an image.
+3.  Removes predicted boxes with either height or width < *min_size*.
+4.  Sorts all `(proposal, score)` pairs by score from highest to lowest; order of pairs with equal scores is undefined.
+5.  Takes top *pre_nms_count* proposals, if total number of proposals is less than *pre_nms_count* takes all proposals.
+6.  Applies non-maximum suppression with *adaptive_nms_threshold*. The initial value of *adaptive_nms_threshold* is
+*nms_threshold*. If `nms_eta < 1` and `adaptive_threshold > 0.5`, update `adaptive_threshold *= nms_eta`.
+7.  Takes top *post_nms_count* proposals and returns these top proposals, scores and the number of proposals. If
+attribute *dynamic_output* is false and total number of proposals is less than *post_nms_count* returns output tensors
+filled with zeroes. If *dynamic_output* is false, then need not to fill output tensors with zeros. No matter attribute
+*dynamic_output* is true or false, the value will be the actual number of proposals.
```

Comment:
No need for this attribute. Let's follow the same rule of NMS and nonzero. Always be dynamic.

Reply:
nms is not always be dynamic. For example, https://github.com/openvinotoolkit/openvino/blob/78db56d92d5fbff0fad93c03c486341cef827718/src/plugins/intel_cpu/src/nodes/mkldnn_multiclass_nms.cpp#L290 . Have discussed with @luo-cheng2021 , it's better to add an attribute here. Besides, it should be compatiable with the original op behavior which is static output.

Reply:
Still not understand. We could make it dynamic once the plugin supports dynamism. There is no need an attribute in spec.

Reply:
It is outdated. If you check the file in new commit. There is no dynamic output attribute.